src/lib.rs

```diff
@@ -37,7 +37,7 @@
 //! use inquire::{Text, validator::{StringValidator, Validation}};
 //!
 //! fn main() {
-//!     let validator: StringValidator = &|input| if input.chars().count() > 140 {
+//!     let validator = |input: &str| if input.chars().count() > 140 {
```

Comment:
Having to put `StringValidator` was something I always disliked. This is much better in the majority of cases :)